experiences/homepage/homepage.test.js

```diff
@@ -77,6 +76,28 @@ const homepageTest = function() {
 		expect(isAtInitialSnapPoint).equal(true);
 	});
 
+	it('card should get promoted at certain element', async() => {
+		await scrollToElement(browser, config.cards.homepage.triggers.myScrollTrigger.spec.selector);
```

Comment:
I think it should be `await scrollToElement(browser, config.cards.homepage.triggers.myIntersectionTrigger.spec.selector);`